GameData/RealismOverhaul/Engine_Configs/Aestus_Config.cfg

```diff
@@ -1,65 +1,178 @@
-//Aestus
-// AIES
+//  ==================================================
+//  Aestus & Aestus II global engine configuration.
+
+//  Inert Mass: 111 Kg (Aestus), 138 Kg (Aestus II)
+//  Throttle Range: N/A
+//  O/F Ratio: 1.9
+//  Burn Time: 1100 s (Aestus), 600 s (Aestus II)
+
+//  Sources:
+
+//  Norbert Brügge - Ariane 5 launch vehicle: http://www.b14643.de/Spacerockets_1/West_Europe/Ariane-5/Description/Frame.htm
+//  Norbert Brügge - European rocket engines: http://www.b14643.de/Spacerockets/Specials/European_Rocket_engines/engines.htm
+//  Encyclopedia Astronautica - Aestus:       http://www.astronautix.com/a/aestus.html
+//  Encyclopedia Astronautica - Aestus II:    http://www.astronautix.com/a/aestus-2.html
+
+//  Used by:
+
+//  * AIES
+//  * RealFuels
+
+//  Notes:
+
+//  * The Aestus only ever had a partial failure on Flight L510 on 31 flights.
+//  * Reusing the Aestus reliability data for the Aestus II engine.
+//  ==================================================
+
 @PART[*]:HAS[#engineType[Aestus]]:FOR[RealismOverhaulEngines]
 {
-	%title = Aestus Vacuum Engine
+	%category = Engine
+	%title = Aestus Series
 	%manufacturer = EADS Astrium
-	%description = Upper stage engine of the Ariane 5ES. Burns hypergolic propellants. Diameter: [1.5 m].
-	
+	%description = A hypergolic upper stage engine. The baseline Aestus engine is pressure fed and it used on the Ariane 5 G, GS and ES series, while the Aestus II (also known as RS-72) variant is pump-fed and develops a higher overall performance. Diameter: 1.32 m.
+
+	@MODULE[ModuleEngines*]
+	{
+		%minThrust = 30.0
+		%maxThrust = 30.0
+		%heatProduction = 36
+		@useEngineResponseTime = False
+		@useThrustCurve = False
+		%EngineType = LiquidFuel
+		%ullage = True
+		%pressureFed = True
+		%ignitions = 5
```

Comment:
What meaningfully limits the number of ignitions on this engine?(pressure fed hypergolic, should be easy to restart usually) my sources simply say "multiple ignitions" no suggestion of a limit at 5.